console/console-init/ui/src/modules/msg-and-iot/components/ProjectList/ProjectList.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import ReactDom from "react-dom";
+import { MemoryRouter } from "react-router";
+import { render, cleanup } from "@testing-library/react";
+import { StatusTypes, ProjectTypes } from "modules/msg-and-iot/utils";
+import { ProjectList, IProject } from "./ProjectList";
+
+afterEach(cleanup);
+
+describe("<ProjectList/>", () => {
+  const projects: IProject[] = [
+    {
+      projectType: ProjectTypes.MESSAGING,
+      name: "namespace_test1.new_space",
+      displayName: "new_space",
+      namespace: "namespace_test1",
+      plan: "Standard",
+      status: StatusTypes.FAILED,
+      creationTimestamp: "2020-01-20T11:44:28.607Z",
+      errorMessageRate: 3,
+      addressCount: 15,
+      connectionCount: 3
+    },
+    {
+      projectType: ProjectTypes.IOT,
+      name: "devops_jbosstest1.k8s_iot",
+      displayName: "k8s_iot",
+      namespace: "devops_jbosstest1",
+      status: StatusTypes.ACTIVE,
+      creationTimestamp: "2020-05-21T14:40:28.607Z",
+      errorMessageRate: 25,
+      deviceCount: 10500,
+      activeCount: 7100
+    }
+  ];
+  it("should render without crashing", () => {
```

Comment:
As discussed, please include a test case for null checks.

Reply:
done